bears/csv/CSVLintBear.py

```diff
@@ -9,6 +9,15 @@
 class CSVLintBear:  # pragma nt: no cover
     """
     Verifies using ``csvlint`` if ``.csv`` files are valid CSV or not.
+    The ``CSVLintBear`` has validation that checks:
+    - structural formatting of a CSV file
+    - delimiter-separated values (dsv) file accessible via URL, File,
+    or an IO-style object (e.g. StringIO)
+    - against CSV dialects
+    - against multiple schema standards; CSV on the Web
+
+    Find out more at:
+    <https://github.com/theodi/csvlint.rb/blob/master/README.md>
```

Comment:
Use `SEE_MORE`.

Reply:
@bkhanale so you want me to replace "Find out more at:" with "SEE_MORE" followed by the link?

Reply:
`git grep SEE_MORE`. You can find lot of examples.

Reply:
Got it! Thanks for the review.